README.md

```diff
@@ -23,6 +23,95 @@ The documentation for various Merge category API's can be found here:
 * [HRIS](https://www.merge.dev/docs/hris/overview/) (Human Resource Information Systems)
 * [Ticketing](https://www.merge.dev/docs/ticketing/overview/)
 
+### Basic Usage
+
+Common to all categories is the Merge SDK `Configuration` object, which defines the authentication properties of 
+your requests. The `apiKey` property is the API Key of your Merge account, which can be administered in the 
+[Merge dashboard](https://app.merge.dev/configuration/keys). The `accessToken` property will be the 
+`X-Account-Token` header which identifies which of your customers' data you are requesting. See the following
+examples for calling various category API's:
+
+#### Accounting
+
+```
+const confAccounting = new Configuration({
+    apiKey: "REDACTED-YourAPIKeyWithMerge",
+    accessToken: "REDACTED-YourCustomer1Key"
+});
+
+const acctApi = new merge_sdk.Accounting.AccountsApi(confAccounting);
+
+// lists the accounting accounts for customer 1
+let response = await acctApi.accountsList({}).catch((reason) => {
```

Comment:
nit pt 2: `const`s for these `let`s

and another nit: skip the abbreviations lol, imo makes it harder to read

Reply:
eh i think i'll leave the let on the response but you're right on the variable naming, shouldn't cut corners.